app/error.jsx

```
@@ -0,0 +1,30 @@
+'use client';
+
+import { useEffect } from 'react';
+import styles from './error.module.css';
+
+export default function Error({ error, reset }) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)